### PR TITLE
Update EIP-5750: Move to Last Call

### PR DESCRIPTION
**Pull request by xinbenlv**

### CODE REVIEW

**Comment by Pandapip1** [comment]
There are a few typos. I'll submit a PR to fix them.

**Comment by xinbenlv** [comment]
Holding this in favor of #5866. will resume when addressed comments

**Comment by xinbenlv** [comment]
request for review @Pandapip1

**Review by Pandapip1 — APPROVED** [review]

File: EIPS/eip-5750.md [hunk 1]
```diff
@@ -21,7 +22,7 @@ The purpose of having extra data in a method is to allow further extensions to e
 
 It is it useful to make methods extendable. Any methods complying with this EIP, such as overloaded `transfer` and `vote` could use string reasons as the extra data. Existing EIPs that have exported methods compliant with this EIP can be extended for behaviors such as using the extra data to prove endorsement, as a salt, as a nonce, or as a commitment for a reveal/commit scheme. Finally, data can be passed forward to callbacks.
 
-There are two ways to achieve extensibility for existing functions. Each comes with their set of challenges:
+There are two ways to achieve extensibility for existing functions. Each comes with a their set of challenges:
```
Comment (Pandapip1) [inline EIPS/eip-5750.md:25]:
```suggestion
There are two ways to achieve extensibility for existing functions. Each comes with their set of challenges:
```

**Review by Pandapip1 — CHANGES_REQUESTED** [review]
Wait never mind - typo

**Review by Pandapip1 — APPROVED** [review]
Now we're good 😅

**Pull request merged**